### DISCUSSION TITLE
Cannot search any repo?

### DISCUSSION THREAD

**Discussion by DynamicLoader** [community/community#145224] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Search 'kernel' returns 45M code and **0 repo**??
![{20F58127-CFA0-4B0A-8163-3ABFF76E68D9}](https://github.com/user-attachments/assets/d90cc649-1878-4ea7-9bbd-3187b34bcd0b)

**Comment by TheJaredWilcurt** [comment]
Also searching for repos on an organization page yields no results. Example:

* https://github.com/freeCodeCamp

**Comment by TheJaredWilcurt** [comment, marked as answer]
https://www.githubstatus.com/incidents/dnq4lp93t62f
